### PR TITLE
Landing page improvements

### PR DESCRIPTION
**Pull request by icarusz**
I will work with Stephanie to fine tune the wording and am open to suggestions but figure this is a good step... so want to get it pushed up ASAP and we can continue to iterate.
I will also take another pass at the chaintree diagram as a near term iteration as well.

### CODE REVIEW

**Comment by icarusz** [comment]
Screen Shot 2019-10-28 at 5 05 05 PM

File: index.md [hunk 1]
```diff
@@ -7,60 +7,46 @@ permalink:
...
+The WASM SDK enables you to easily leverage the power of trust that Tupelo
+makes possible right from a browser.
+{: .fs-3 .fw-300 }
 
+Another option is to run through our [“Hello World” tutorials](tutorials) and have your
+first transaction signed in minutes. That is a total of minutes to set up and submit the
+request, the Tupelo TestNet will sign your transaction in less than a second.
+{: .fs-3 .fw-300 }
+***
 # We Are Distributed Ledger Done Differently
```
Comment (tobowers) [inline index.md]: Tupelo is Distributed Ledger Done Differently

**Review by tobowers — APPROVED** [review]
looks so much better.

File: index.md [hunk 2]
```diff
@@ -7,60 +7,46 @@ permalink:
 
 # Hello Decentralized World
 {: .fs-8 }
-<img style="float: right; padding:10px;" src="assets/images/helloworldsquare.png">
+<img style="float:right; padding:6px; width:529px;" src="assets/images/wasm_sample_code.png">
```
Comment (tobowers) [inline index.md:10]: I think ideally this would not be an image, but rather a div with a `<code>` tag in it.

Comment (icarusz) [reply index.md:10]: Agreed.  It would be better as code and then even "cut and pasteable".
I will look into that.

File: index.md (hunk 1 above)
Comment (icarusz) [reply index.md]: Doh, missed in title.  Fixorated.

**Pull request merged**